### DISCUSSION TITLE
Issue with Dynamic Prompts wildcards

### DISCUSSION THREAD

**Discussion by temporaryspastic** [adieyal/sd-dynamic-prompts#568] [Q&A] [answered]
So I had asked this question before but I didn't get any clear answers, but I figure maybe I didn't explain my issue well enough so here goes round 2 lol.

Anytime I use a wildcard entry by itself without any other words in my prompt, it will give me a result from the text list.  Here is an example of using the prompt "bird".  Please note the image title.

![bird1_red](https://github.com/adieyal/sd-dynamic-prompts/assets/139179959/c5599cc3-36f2-43bf-a32e-3813e27e26c0)

Now, if I added any other modifier to my prompt, even a single letter, it seemingly neglects the wildcard completely and just creates a prompt based off the words and does not utilize the text file.  Below are 2 examples.  One is with the word "yellow" added prior to the wildcard entry, and the other is using another wildcard in the prompt.

![bird2_red](https://github.com/adieyal/sd-dynamic-prompts/assets/139179959/308c26fe-3c6c-4d33-bd39-85f27d45ed5b)

![bird3_red](https://github.com/adieyal/sd-dynamic-prompts/assets/139179959/556a0292-a74b-4161-bfd8-911517bd7f51)

Here are the extensions I am using.  I have tried deactivating some of the other extensions to see if they are causing a problem, but it seems that the "stable-diffusion-webui-wildcards" extension needs to be activated with or without the "sd-dynamic-prompts" extension to work with even the solitary wildcard prompt.  Interestingly enough this was an older extension that I used that I don't see anymore on the stock github extension repository. Also, my extensions are saved in "C:\stable-diffusion-automatic 1111\stable-diffusion-webui\extensions", which I believe is correct.

![extension list](https://github.com/adieyal/sd-dynamic-prompts/assets/139179959/94009750-85bb-469e-904a-54fe2abc3be8)

Any help would be greatly appreciated! Thanks in advance.

**Comment by adieyal** [comment, marked as answer]
do you get any errors in the console?

**Reply by temporaryspastic** [reply]
Now that you mention it, I do see it something on the console that says "File C:\stable-diffusion-automatic 1111\stable-diffusion-webui\wildcards\bird.txt not found for the __bird__ wildcard"

![console](https://github.com/adieyal/sd-dynamic-prompts/assets/139179959/358b3bc7-a46a-4c29-afc2-2eae828a0ff5)

I guess I needed a wildcard folder in that directory and not just within the extensions directory.  I just copied and pasted the folder to that directory and it works! Thanks for reminding me to check the console!

![success](https://github.com/adieyal/sd-dynamic-prompts/assets/139179959/9e400254-fecf-4865-a583-73543bf633d3)

**Reply by adieyal** [reply]
strange - did you configure the wildcards directory to point to `C:\stable-diffusion-automatic 1111\stable-diffusion-webui\wildcards\`
